polish_labels.json

```diff
@@ -0,0 +1,62 @@
+{
+    "alarm_query": "alarm_zapytanie",
+    "alarm_remove": "alarm_usuwanie",
+    "alarm_set": "alarm_ustawianie",
+    "audio_volume_down": "dźwięk_ciszej",
```

Comment:
Maybe I should leave `'audio'` or change it to `'głośność'` (but it is even longer than `'dźwięk'` and has more Polish characters)?